DEVELOPMENT.md

```diff
@@ -136,31 +136,32 @@ Kubernetes cluster in your designated environment, if necessary.
 ### Deploy Istio
 
 ```shell
-STABLE_VERSION=$(curl https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/istio-stable)
-kubectl apply -f "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${STABLE_VERSION}/istio-crds.yaml"
-while [[ $(kubectl get crd gateways.networking.istio.io -o jsonpath='{.status.conditions[?(@.type=="Established")].status}') != 'True' ]]; do
-  echo "Waiting on Istio CRDs"; sleep 1
-done
-kubectl apply -f "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${STABLE_VERSION}/istio-minimal.yaml"
-kubectl apply -f ./third_party/net-istio.yaml
+LATEST_VERSION=$(curl https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/istio-latest)
+wget "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${LATEST_VERSION}/install-istio.sh"
```

Comment:
Wondering if there's a reason we're downloading and executing in two steps rather than using something like `curl -sSL https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${LATEST_VERSION}/install-istio.sh | sh` here?

I'm also wondering a little, since these are development instructions, if we should just run this from a vendored copy in `third_party` like we did before with net-istio.yaml? Or is this part of an effort to decouple the repos?

Reply:
Piping to `sh` won't work because you need to pass a profile as parameter and it needs to be downloaded as well.
Of course, it could be done with `xargs` but that's beside the point.
Instead of vendored copy, why not clone the `net-istio` repo? That's what E2E scripts do, and it makes sense for development.

Reply:
yeah I was wondering about just asking people to clone too, that'd work. 

Re: passing a profile as a parameter, fwiw looking at the script it already [goes and downloads Istio from a url](https://github.com/knative-sandbox/net-istio/blob/master/third_party/istio-1.5.7/install-istio.sh#L20), I don't see why it can't also manage pulling down the manifest based on a flag instead of expecting it to be local?

Reply:
I'd agree with Julz on expecting the repo to be cloned for development instructions.

Reply:
OK. I followed the suggestion here to clone net-istio repo first and then install Istio from net-istio repo.